.github/workflows/nodejs.yml

```diff
@@ -0,0 +1,64 @@
+name: CI Build
+
+on: [push]
```

Comment:
Need to add more things here, at least `pull_request` (or it won't run on PR's from a fork)

Reply:
Added `pull_request`.